packages/opentelemetry-exporter-collector-grpc/src/CollectorExporterNodeBase.ts

```diff
@@ -41,13 +42,16 @@ export abstract class CollectorExporterNodeBase<
   grpcQueue: GRPCQueueItem<ExportItem>[] = [];
   metadata?: Metadata;
   serviceClient?: ServiceClient = undefined;
+  serverAddress!: string;
```

Comment:
```suggestion
  serverAddress: string;
```
As it is assigned in constructor it should be not needed